examples/calculator/src/calculator_android.nit

```diff
@@ -82,7 +84,33 @@ redef class App
 			end
 		end
 
-		context.content_view = layout
+		native.content_view = layout
+	end
+
+	redef fun on_save_instance_state(state)
+	do
+		super
+
+		var nity = new Bundle
+		nity["context"] = context.to_json
+	end
+
+	redef fun on_restore_instance_state(state)
+	do
+		super
+
+		var nity = new Bundle
+		if not nity.has("context") then return
+
+		var json = nity.string("context")
+		if json == null then return
+
+		context = new CalculatorContext.from_json(json)
+	end
+
+	redef fun on_resume
+	do
+		display.text = context.display_text
```

Comment:
Is there a link between the context here and the context that you save and restore in the two previous methods?